### PR TITLE
feat(input-time-picker): deprecate active prop and add open prop

### PR DESCRIPTION
**Pull request by anveshmekala**
**Related Issue:** #4688 

## Summary

This PR will deprecate the `active` prop and adds support for the `open` prop in `input-time-picker`.

### CODE REVIEW

**Review by driskull — APPROVED** [review]
LGTM!

Any stories to update?

**Comment by anveshmekala** [comment]
> LGTM!
> 
> Any stories to update?

Nope, none of the stories are using `active` prop. Should we add one ?

**Comment by driskull** [comment]
It would be a nice touch to add one. thanks! 💯

File: src/components/input-time-picker/input-time-picker.stories.ts [hunk 1]
```diff
@@ -51,6 +51,15 @@ export const Placement = (): string => html`
   </calcite-input-time-picker>
 `;
 
+export const Open = (): string => html`
```
Comment (driskull) [inline src/components/input-time-picker/input-time-picker.stories.ts:54]: nice extra credit! ☝️

File: src/components/input-time-picker/input-time-picker.tsx [hunk 2]
```diff
@@ -46,13 +46,27 @@ export class InputTimePicker implements LabelableComponent, FormComponent, Inter
...
+   */
   @Prop({ reflect: true, mutable: true }) active = false;
 
   @Watch("active")
-  activeHandler(): void {
+  activeHandler(value: boolean): void {
+    this.open = value;
+  }
+
+  /** When tru, opens the `time-picker` component */
```
Comment (jcfranco) [inline src/components/input-time-picker/input-time-picker.tsx:61]: Typo: `tru` => `true`. 

@benelan @geospatialem Doc-wise, would you recommend having the description be generic? `open` is used across different components.

**Review by jcfranco — APPROVED** [review]
💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥
💥💣💣💣💣💥💥💣💣💥💥💥💣💣💥💥💣💥💥💥💣💥
💥💣💥💥💣💥💣💥💥💣💥💣💥💥💣💥💣💣💥💣💣💥
💥💣💣💣💥💥💣💥💥💣💥💣💥💥💣💥💣💥💣💥💣💥
💥💣💥💥💣💥💣💥💥💣💥💣💥💥💣💥💣💥💥💥💣💥
💥💣💣💣💣💥💥💣💣💥💥💥💣💣💥💥💣💥💥💥💣💥
💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥

Good to merge once the typo is fixed.

File: src/components/input-time-picker/input-time-picker.tsx (hunk 2 above)
Comment (geospatialem) [reply src/components/input-time-picker/input-time-picker.tsx:61]: Good catch for doc consistency, @jcfranco! 

@anveshmekala, Would the statement below be in line with the `input-time-picker` component functionality? 

> When true, displays and positions the component. 

We've been leveraging the above line for other `open` props across components, as long as the context makes sense.

Comment (anveshmekala) [reply src/components/input-time-picker/input-time-picker.tsx:61]: I agree it's a bit confusing. how about  _When true, opens the `time-picker` component_  ?

Comment (geospatialem) [reply src/components/input-time-picker/input-time-picker.tsx:61]: Ah, my mistake (lol, about to sign off here) - thanks for clarifying, Anvesh. 

For consistency, can we add `calcite` to the component name and a period at the end?

> When true, opens the `calcite-time-picker` component.

Comment (anveshmekala) [reply src/components/input-time-picker/input-time-picker.tsx:61]: sure.

**Pull request merged**